client/src/pages/Login/Login.js

```diff
@@ -15,10 +15,10 @@ export default class Login extends Component {
       baseUrl: process.env.REACT_APP_ISSUER.split('/oauth2')[0],
       clientId: process.env.REACT_APP_CLIENT_ID,
       redirectUri: process.env.REACT_APP_REDIRECT_URI,
-      logo: '/react.svg',
+      logo: '/logo.svg',
       i18n: {
         en: {
-          'primaryauth.title': 'Sign in to React & Company',
+          'primaryauth.title': 'Orator',
```

Comment:
So we are going back to Orator? lol. Ill make sure to update on the server.js file